docs/testing/ie-11-testing.md

```diff
@@ -1,44 +1,50 @@
 ---
-ms.date: 05/16/2020
-description: 'Test your Office Add-in using Internet Explorer 11.'
 title: Internet Explorer 11 testing
+description: 'Test your Office Add-in on Internet Explorer 11.'
+ms.date: 05/19/2021
 localization_priority: Normal
 ---
 
-# Test your Office Add-in using Internet Explorer 11
+# Test your Office Add-in on Internet Explorer 11
 
-Depending on the specifications of your add-in, you may plan to support older versions of Windows and Office, which require testing on Internet Explorer 11. This is often necessary as part of submitting your add-in to AppSource. You can use the following command line tooling to switch from more modern runtimes used by add-ins to the Internet Explorer 11 runtime for this testing.
+If you plan to market your add-in in AppSource or you plan to support certain older versions of Windows and Office, you must test the add-in using the embeddable web view control that is based on Internet Explorer 11 (called "Trident"). You can use a command line to switch from more modern runtimes used by add-ins to the Internet Explorer 11 runtime for this testing. For information about which versions of Windows and Office use the Internet Explorer 11 web view control, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

Comment:
Here's the same text I suggested in an earlier comment to avoid "Trident":
```suggestion
If you plan to market your add-in through AppSource or you plan to support older versions of Windows and Office, your add-in must work in the embeddable browser control that is based on Internet Explorer 11 (IE11). You can use a command line to switch from more modern runtimes used by add-ins to the Internet Explorer 11 runtime for this testing. For information about which versions of Windows and Office use the Internet Explorer 11 web view control, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```